.buildkite/longruns/pipeline.yml

```diff
@@ -300,11 +299,11 @@ steps:
 
   # DYAMOND AMIP: 1 day (convection resolving)
   - label: "GPU AMIP SUPERFINE: dyamond_target"
-    key: "gpu_dyamond_target"
+    key: "gpu_longrun_amip_dyamond"
```

Comment:
I renamed this to be consistent with the atmos equivalent ([longrun_aquaplanet_dyamond.yml](https://github.com/CliMA/ClimaAtmos.jl/blob/main/config/longrun_configs/longrun_aquaplanet_dyamond.yml))